src/main/java/uk/gov/hmcts/reform/sscs/exception/AuthorisationException.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.hmcts.reform.sscs.exception;
+
+@SuppressWarnings("squid:MaximumInheritanceDepth")
+public class AuthorisationException extends RuntimeException {
```

Comment:
Weren't we going to make these checked exceptions to ensure they're caught and handled properly?